src/components/presentational/SingleSurveyTask/SingleSurveyTask.tsx

```diff
@@ -17,75 +16,98 @@ import { ButtonVariant } from '../Button/Button';
 import checkMark from '../../../assets/greenCheck.svg';
 import Action from '../Action';
 import LoadingIndicator from '../LoadingIndicator';
+import { getLocale } from '../../../helpers/date-helpers';
+import { noop } from '../../../helpers/functions';
+import UnstyledButton from '../UnstyledButton';
+
+export type SingleSurveyTaskVariant = 'default' | 'expanded';
 
 export interface SingleSurveyTaskProps {
-	task: SurveyTask,
-	descriptionIcon?: IconDefinition,
+	task: SurveyTask;
 	onClick: () => void;
+	variant?: SingleSurveyTaskVariant;
+	descriptionIcon?: IconDefinition;
 	surveyActive?: boolean;
-	innerRef?: React.Ref<HTMLDivElement>;
 	buttonColor?: ColorDefinition;
 	buttonVariant?: ButtonVariant;
+	innerRef?: React.Ref<HTMLDivElement>;
 }
 
 export default function (props: SingleSurveyTaskProps) {
-	const context = useContext(LayoutContext);
-
-	const datesAreOnSameDay = (first: Date, second: Date) =>
-		first.getFullYear() === second.getFullYear() &&
-		first.getMonth() === second.getMonth() &&
-		first.getDate() === second.getDate();
+	const layoutContext = useContext(LayoutContext);
 
-	let today = new Date();
-	let tomorrow = add(new Date(), { days: 1 });
-	let dueDateString = "";
-	let dueDateIntent = "";
-	let locale = MyDataHelps.getCurrentLanguage().toLowerCase().startsWith("es") ? es : enUS;
-	if (props.task.dueDate) {
+	const getDueDate = () => {
+		let today = startOfToday();
+		let tomorrow = add(new Date(today), {days: 1});
```

Comment:
nit: startOfTomorrow

Reply:
I'm not sure it needs to be that specific.  The "start of" is just being used to remove the fluctuating time component from both dates since that part isn't really relevant for the "same day/next day" logic being applied here.